src/components/colorPicker/colorPicker.api.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ColorPicker",
+  "category": "form",
+  "description": "A picker component for color selection",
+  "notes": "This is a screen width component",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ColorPickerScreen.tsx",
+  "images": [
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/ColorPicker/ColorPicker.gif?raw=true"
+  ],
+  "props": [
+    {
+      "name": "colors",
+      "type": "string[]",
+      "description": "Array of colors for the picker's color palette (hex values)"
+    },
+    {"name": "value", "type": "string", "description": "The value of the selected swatch"},
+    {
+      "name": "animatedIndex",
+      "type": "number",
+      "description": "The index of the item to animate at first render",
+      "note": "Default is last"
+    },
+    {
+      "name": "onValueChange",
+      "type": "(value: string, options: object) => void",
+      "description": "Callback for the picker's color palette change"
+    },
+    {
+      "name": "accessibilityLabels",
+      "type": "{\n  addButton?: string,\n  dismissButton?: string,\n  doneButton?: string,\n  input?: string}",
+      "description": "Accessibility labels as an object of strings",
+      "default": "{\n  addButton: 'add custom color using hex code',\n  dismissButton: 'dismiss',\n  doneButton: 'done',\n  input: 'custom hex color code'\n}"
```

Comment:
Will it break the line in `.md` file?

Reply:
I hope

Reply:
I think the `<br />` will do the work here

Reply:
What's the difference and why use markup in a string?

Reply:
I tried to play with a `.md` file and I saw that `\n` doesn't break the line..

Reply:
ok... We are using \n very often, did you run the script and see it doesn't break the lines on the doc site?

Reply:
Nope, Ethan said it makes `.md` files from the api files so I guess it'll work the same..
but you can check if it works that way and keep it.

Reply:
Yes, I saw it in the script. I'll check

Reply:
\n is not working but not visible, while <br/> is not working and it is visible in the string

Reply:
I'll leave the \n till we find a solution so we'll know where we need a fix